app/presenters/admin_email_list.rb

```diff
@@ -123,8 +129,8 @@ def artist_as_csv_row(artist)
       artist.get_name(escape: false),
       artist.email,
       artist.studio ? artist.studio.name : '',
-    ] + available_open_studios_keys.map do |os_tag|
-      artist.doing_open_studios?(os_tag).to_s
+    ] + available_open_studios_events.map do |os_event|
+      os_event.open_studios_participants.exists?(user: artist)
```

Comment:
This could likely be optimized from the query perspective but it's admin so... who cares (for now)